wizards/openacademy_wizard.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields, api
```

Comment:
It is good not to make imports that we do not use in the code, this to avoid Lint problems.

I know that later you can end up using it, but since this is not the case at the moment it could be removed.

```suggestion
from odoo import models, fields
```

Reply:
Got it.